crates/superposition_types/src/lib.rs

```diff
@@ -187,6 +187,32 @@ pub struct PaginatedResponse<T> {
     pub data: Vec<T>,
 }
 
+#[derive(
+    Debug, Serialize, Deserialize, Clone, PartialEq, PartialOrd, derive_more::Display,
+)]
+#[serde(rename_all = "lowercase")]
+pub enum SortBy {
+    #[display(fmt = "desc")]
+    Desc,
+    #[display(fmt = "asc")]
+    Asc,
+}
+
+impl SortBy {
+    pub fn flip(&self) -> Self {
+        match self {
+            SortBy::Desc => SortBy::Asc,
+            SortBy::Asc => SortBy::Desc,
+        }
+    }
+}
+
+impl Default for SortBy {
+    fn default() -> Self {
+        Self::Desc
```

Comment:
I feel default should be Asc

Reply:
@pratikmishra356 you always want to see the latest data, especially if you've been using Superposition for years.

Reply:
@Datron but it can differ based on entity , position/weight needs to be sort in asc order

Reply:
You can set it in the sort function based on needs, don't need to call default